src/access/AccessManagerEnumerable.sol

```diff
@@ -73,13 +122,48 @@ contract AccessManagerEnumerable is AccessManager, IAccessManagerEnumerable {
     return targetFunctionSelectors;
   }
 
+  /// @dev Tracks all role identifiers when a new role is created.
+  function _trackRole(uint64 roleId) internal {
+    if (!_rolesSet.contains(uint256(roleId))) {
+      _rolesSet.add(uint256(roleId));
+    }
```

Comment:
`add` already has this logic https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v5.4.0/contracts/utils/structs/EnumerableSet.sol#L74-L84 we just need to ignore the return the value